app/editor/components/FindAndReplace.tsx

```diff
@@ -278,7 +288,10 @@ export default function FindAndReplace({
         delay={500}
         placement="bottom"
       >
-        <ButtonLarge onClick={() => editor.commands.prevSearchMatch()}>
+        <ButtonLarge
+          disabled={totalResults === 0}
```

Comment:
Disable the navigation buttons if there are no results

Reply:
Disable the replace / replaceAll buttons as well?

While we're here, do you think adding a close button would be beneficial?

Reply:
Not going to add a close button right now as there is already an abundance of controls in this small space and there  are two ways to close with `cmd+f` and `esc`